cg/models/taxprofiler/taxprofiler.py

```diff
@@ -40,7 +41,7 @@ class TaxprofilerParameters(WorkflowParameters):
     centrifuge_save_reads: bool = True
     run_krona: bool = True
     run_profile_standardisation: bool = True
-    clusterOptions: str = Field(..., alias="cluster_options")
+    clusterOptions: str = Field(str, alias="cluster_options")
```

Comment:
```suggestion
    clusterOptions: str = Field(alias="cluster_options")
```
I do not think you need the type hint in the Field instantiation at all